docs/install/iso-install.md

```diff
@@ -28,30 +28,62 @@ Note: This [video](https://youtu.be/X0VIGZ_lExQ) shows a quick overview of the I
 
 1. Mount the Harvester ISO disk and boot the server by selecting the `Harvester Installer` option.
    ![iso-install.png](/img/v1.2/install/iso-install.png)
-1. Choose the installation mode by either creating a new Harvester cluster, or by joining an existing one.
+
+1. Choose an installation mode: 
+   - Create a new Harvester cluster.
       - Note: By default, the first node will be the management node of the cluster. When there are 3 nodes, the other 2 nodes added first are automatically promoted to management nodes to form an HA cluster.
-      
+
       If you want to promote management nodes from different zones, you can add the node label `topology.kubernetes.io/zone` in the [os.labels](./harvester-configuration.md#oslabels) config by providing a URL of [Harvester configuration](./harvester-configuration.md) on the customize the host step. In this case, at least three different zones are required.
-1. Choose the installation device on which the Harvester cluster will be installed
+
+   - Join an existing Harvester cluster.
+
+   - Install Harvester binaries only.
+      - Note: If you choose `Install Harvester binaries only`, additional setup is required after the first bootup.      
+
+1. Choose the installation device on which the Harvester cluster will be installed.
+   ![select-disk.png](/img/v1.2/install/select-disk.png)
       - Note: By default, Harvester uses [GPT](https://en.wikipedia.org/wiki/GUID_Partition_Table) partitioning schema for both UEFI and BIOS. If you use the BIOS boot, then you will have the option to select [MBR](https://en.wikipedia.org/wiki/Master_boot_record).
-   ![iso-install-disk.png](/img/v1.2/install/iso-install-disk.png)
-1. We recommend choosing a separate disk to store VM data.
-   ![iso-install-disk.png](/img/v1.2/install/iso-select-data-disk.png )
-1. Configure the hostname and select the network interface for the management network. By default, Harvester will create a bonded NIC named `mgmt-bo`, and the IP address can be configured via DHCP or a statically assigned one <small>(Note: The Node IP can not change at the lifecycle of a Harvester cluster. If DHCP is used, users must make sure the DHCP server always offers the same IP for the same Node. If Node IP is changed, the related Node cannot join the cluster and might even break the cluster)</small>.
 
-   ![iso-installed.png](/img/v1.2/install/iso-nic-config.png)
+      If you only have one disk, or have to use the same disk for both OS and VM data, you need to configure persistent partition size, which is used to store system packages and container images (the default and minimum value is 150 GB).
```

Comment:
```suggestion
      If you only have one disk, or have to use the same disk for both OS and VM data, you need to configure persistent partition size, which is used to store system packages and container images (the default and minimum value is 150 GiB).
```

Reply:
Do we need to change all `GB` to `Gi` or `GiB`?